drivers/serial/Kconfig.cdns

```diff
@@ -0,0 +1,9 @@
+#Copyright Meta Platforms, Inc.and its affiliates.
+#SPDX-License-Identifier: Apache-2.0
+
```

Comment:
The DT_COMPAT_** needs to be added within here for this to enable the kconfig from the dts

Reply:
Updated. Thanks.